docs/docs/reference/metaprogramming/tasty-reflect.md

```diff
@@ -26,7 +26,7 @@ the scope where it is used.
 ```scala
 import scala.quoted._
 
-inline def natConst(x: => Int): Int = ${natConstImpl('{x})}
+inline def natConst(inline x: Int): Int = ${natConstImpl('{x})}
```

Comment:
Might be good to change the title of the doc to `AST Reflect`, as it's done in the code to avoid obscure terms to programmers and it is more accurate.

Reply:
I will do it in the next docs update PR